ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -197,9 +194,6 @@ class MSFCardNudgeStateImpl: NSObject, ControlConfiguration, MSFCardNudgeState {
     /// Action to be dispatched by the dismiss ("close") button on the trailing edge of the control.
     @Published var dismissButtonAction: CardNudgeButtonAction?
 
-    /// Design token set for this control, to use in place of the control's default Fluent tokens.
-    @Published var overrideTokens: CardNudgeTokens?
```

Comment:
Just to help me understand better: What's the reasoning behind moving the "override tokens" (now token resolver) out of the state object?

Reply:
Simplicity. We have to introduce a new class that does token stuff (`TokenResolver`), so it's simplest from a conceptual point to just let that class also own the truth of the current `overrideTokens`, instead of trying to keep the two in sync.